common/logger/src/lib.rs

```diff
@@ -138,11 +139,14 @@ pub use text_log::{Logger, CHANNEL_SIZE, DEFAULT_TARGET};
 pub mod counters;
 
 use serde::Serialize;
+use std::str::FromStr;
+
+static LOG_LEVEL_NAMES: [&str; 7] = ["OFF", "CRITICAL", "ERROR", "WARN", "INFO", "DEBUG", "TRACE"];
 
 // severity level - lower is worse
-// TODO: We can remove this if we get rid of critical
-// TODO: should we serialize it into text?
-#[derive(Serialize, Clone, Copy, Debug)]
+// TODO: Consider removing Critical, and using RustLog's level instead, as setting the log level
```

Comment:
let's get rid of crit... I don't want to be a special snowflake!

Reply:
Done